netlify.toml

```
@@ -0,0 +1,10 @@
+[build]
+    fucntions = "netlify/functions"
```

there is a typo

```suggestion
[build]
    functions = "netlify/functions"
```

also do we need this? I think this should be the default?